### PR TITLE
Updated version

### PR DESCRIPTION
**Pull request by Sebastvin**
Updated package for wagtail 5

### CODE REVIEW

File: wagtailcharcount/static/hallo_charcount.js [hunk 1]
```diff
@@ -1,107 +1,88 @@
...
-				}
-			}
-		});
-	})
-
-
-});
+function initializeCharacterCounter() {
+    // Locate the main editor container
+    let editorWrapper = document.querySelector('.w-field--draftail_rich_text_area');
```
Comment (seb-b) [inline wagtailcharcount/static/hallo_charcount.js]: Does this work with multiple richtext fields on the same page?

File: setup.py [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
-with open('README.rst', 'r') as f:
+with open('README.md', 'r') as f:
     readme = f.read()
 
 from setuptools import find_packages, setup
 
 setup(
     name='wagtail-charcount',
-    version='0.1.2',
+    version='0.1.3',
```
Comment (seb-b) [inline setup.py]: can you revert this? We'll handle versioning. This will be a major release not a bugfix

**Comment by Sebastvin** [comment]
:white_check_mark: Revert version
:white_check_mark: Counter work with multiple richtext fields on the same page
image

**Comment by seb-b** [comment]
released in version 0.2.0

**Pull request merged**